eth/block_from_raw.go

```diff
@@ -33,8 +33,10 @@ func (b *Block) FromRaw(input string) error {
 	hash, err := NewHash(h)
 
 	header, txs, uncles := decoded.List[0].List, decoded.List[1].List, decoded.List[2].List
-	// header should be 15 items
-	if len(header) != 15 {
+	// header should be 15 items for legacy blocks, 16 for EIP-1559 blocks
+	switch len(header) {
+	case 15, 16:
```

Comment:
I've not seen this construct used like this before. Does this short hand a no-op for `case 15, 16` and it does not fall through like other languages?

Reply:
Ya Go has an explicit `fallthrough` keyword, see for example: https://play.golang.org/p/p_QIKattOL